### PR TITLE
Nobid Bid Adapter: add support for alias duration

### PR DESCRIPTION
**Pull request by redaguermas**
## Type of change
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
Added support for alias "duration".

- test parameters for validating bids
```
{
  bidder: 'duration',
  params: {
    // ...
  }
}
```

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:
- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/pull/3102

### CODE REVIEW

**Comment by redaguermas** [comment]
Hello,
Can you please review this PR. The error in the CircleCI is unrelated to my code.

**Comment by ChrisHuie** [comment]
@redaguermas rerean tests and passed but looks like you uploaded several of your ide files with this pr and your package-lock file. Can you please remove all those files and add a test case for the nobid change.

**Comment by robertrmartinez** [comment]
@redaguermas Yeah please remove all of your extra files.

Should just be only two files, the bid adapter, and the associated _spec file testing the change.

package-lock should not be in there either.

File: modules/nobidBidAdapter.js [hunk 1]
```diff
@@ -337,6 +339,10 @@ window.addEventListener('message', function (event) {
 }, false);
 export const spec = {
   code: BIDDER_CODE,
+  gvlid: GVLID,
+  aliases: [
+    { code: 'duration', gvlib: 674 }
```
Comment (bretg) [inline modules/nobidBidAdapter.js:344]: this should be gvlid,not glvlib

File: a [hunk 2]
```diff
@@ -0,0 +1,41 @@
+[16:58:25] Using gulpfile ~/Documents/work/nobid/Prebid.js/gulpfile.js
```
Comment (bretg) [inline a:1]: this file is a mistake. please delete it

**Review by bretg — CHANGES_REQUESTED** [review]
please also remove nobidBidAdapter.js.orig and the changes to package-lock.json

**Comment by redaguermas** [comment]
Hello @bretg ,
Thank you for the feedback. Can you please review the latest changes.
-Regards

**Review by ChrisHuie — APPROVED** [review]

**Pull request merged**